src/common/utxobased/db/makeProcessor.ts

```diff
@@ -1225,13 +1252,14 @@ const fetchTxIdsByBlockHeight = async (
 
   // Fetch transaction IDs
   const result = await tables.txIdsByBlockHeight.query('', fromBlock, toBlock)
+  const cleaner = asString
 
   return (
     result
       // RangeBase returns values with lowest value first
       .reverse()
       // Return array of just the IDs
-      .map(({ [RANGE_ID_KEY]: id }) => id)
+      .map(({ [RANGE_ID_KEY]: id }) => cleaner(id))
```

Comment:
Why not just `asString(id`?

Reply:
I don't like the nesting syntax, e.g. `cleaner(cleaner(cleaner))(something)`, so I put the cleaner declaration on a separate line. Everywhere, no reason to do it, so I changed it.

Reply:
Fixed in: https://github.com/EdgeApp/edge-currency-plugins/pull/151/commits/64988911484b0708bb85979760f61b270b35e194